src/context/Result.h

```diff
@@ -21,15 +21,15 @@ class ResultBuilder;
 // An executor will produce a result.
 class Result final {
 public:
-    static const Result kEmptyResult;
-    static const std::vector<Result> kEmptyResultList;
-
     enum class State : uint8_t {
         kUnExecuted,
         kPartialSuccess,
         kSuccess,
     };
 
+    static const Result& emptyResult();
+    static const std::vector<Result>& emptyResultList();
```

Comment:
To comply with `Status::OK()`, better to capitalize the first character.

Reply:
Thx, Fixed.